shaal/DrupalPod
# Site Install with admin password + interdiff install

# The Problem/Issue/Bug
Drupal websites should be installed with random password.
Users cannot run `interdiff` command in terminal.

## How this PR Solves The Problem
This PR set the site-install in the default script to use 'admin' as admin password.
This PR install `patchutils`, which makes `interdiff` command available.

## Manual Testing Instructions
Using the extension, choose any install profile.
Confirm that the site was installed with username `admin`, password `admin`.

Confirm you can run `interdiff` command in terminal.

## Related Issue Link(s)

## Release/Deployment notes



<a href="https://gitpod.io/#https://github.com/shaal/DrupalPod/pull/14"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
